utilities/keys.js

```diff
@@ -69,18 +65,15 @@ function createStore(mnemonic, password, filename = "key") {
     }
 }
 
-function decryptStore(file, password) {
-    const data = fs.readFileSync(file, { encoding: "utf8", flag: "r" });
-    if (
-        JSON.parse(data).hashpwd === crypto.createHash(passwordHashAlgorithm).update(password).digest("hex")
-    ) {
-        let iv = Buffer.from(JSON.parse(data).iv, "hex");
-        let encryptedText = Buffer.from(JSON.parse(data).crypted, "hex");
+function decryptStore(hashpwd, iv, salt, crypted, password) {
```

Comment:
can't we just put the entire file data as one input?
function decryptStore(fileData, password)

Reply:
Added into one input